textile/fabric_printing/doctype/print_order/print_order.py

```diff
@@ -1201,6 +1201,7 @@ def set_missing_values(source, target):
 		if target.meta.has_field("cost_center") and source.get("cost_center"):
 			target.cost_center = source.get("cost_center")
 
+		target.flags.ignore_permissions = ignore_permissions
```

Comment:
Is this necessary?

Reply:
yes, ignore_permissions flag to be set while get party details.